packages/openai/src/chat.ts

```diff
@@ -279,10 +279,14 @@ export class OpenAIChatModel implements IChatModel<ChatCompletionCreateParams, O
       };
 
       if (message.tool_calls && message.tool_calls.length > 0) {
-        return this.send(modelMessage, {
-          ...options,
-          messages: memory,
-        });
+        if (!options.disableAutomaticFunctionCalling) {
+          return this.send(modelMessage, {
+            ...options,
+            messages: memory,
+          });
+        } else {
+          this._log.debug(`Automatic function calling is disabled, skipping function call (total calls: ${message.tool_calls.length})`);
```

Comment:
```suggestion
          this._log.debug(`Automatic function calling is disabled; skipping function call (total calls: ${message.tool_calls.length})`);
```
What is the intent when listing total calls? Is there a case where the developer would need this number when debugging structured output?

Reply:
yeah they'd need to execute those calls manually if they want to re-use the prompt